packages/core/rpc/ElectronRpcDriver.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-await-in-loop */
+/* eslint-disable no-await-in-loop,@typescript-eslint/no-explicit-any */
```

Comment:
not at the beginning of the file please

Reply:
found one more

Reply:
heh can you have an eslint rule to disable certain invocations of eslint-disable?

Reply:
That is very meta 👽